openwillis-face/src/openwillis/face/face_landmark.py

```diff
@@ -900,6 +900,40 @@ def normalize_face_landmarks(
 
     return norm_df
 
+def adjust_summary_stats(df_summ, skip_interval):
+        """
+        ---------------------------------------------------------------------------------------------------
+        Adjusts the mean and standard deviation columns in the summary dataframe based on the skip interval.
+
+        Parameters:
+        ............
+        df_summ : pandas.DataFrame
+        Summary statistics dataframe.
+        skip_interval : int
+        The number of frames skipped during landmark extraction.
+
+        Returns:
+        ............
+        pandas.DataFrame
+        Adjusted summary statistics dataframe.
+        ---------------------------------------------------------------------------------------------------
+        """
+        if skip_interval<0:
+            #raise exception
+            Rai
```

Comment:
I'm not sure what you want to do here, whether it is raise an exception or return `df_summ`, but I guess there is something missing